src/reactfire.js

```diff
@@ -23,12 +38,20 @@ var ReactFireMixin = {
   /*************/
   /* Creates a binding between Firebase and the inputted bind variable as an array */
   bindAsArray: function(firebaseRef, bindVar, cancelCallback) {
-    this._bind(firebaseRef, bindVar, cancelCallback, true);
+    if (!this.setState){
```

Comment:
What exactly is this check ensuring? It seems a bit of a hacky solution. Are there examples of using static methods like this in the React docs that I can take a look at?